### ISSUE TITLE
JWT values in url_pattern do not work for query string values

### ISSUE THREAD

**Issue by MrSmoke** [devopsfaith/krakend#428]
**Describe the bug**
Using JWT values in the `url_pattern` does not work for query string values

**To Reproduce**
```json
"url_pattern": "/__debug/test?userId={JWT.sub}",
````

**Expected behavior**
the sub should be injected

**Logs**
```
[GIN] 2020/11/16 - 07:15:23 | 404 |         1.6µs |      172.17.0.1 | GET      "/__debug/test?userId={{.JWT.sub}}"
[GIN] 2020/11/16 - 07:15:23 | 500 |    361.0658ms |      172.17.0.1 | GET      "/test"
```

**Additional context**
Interestingly enough it seems to work if you put a `/` before the value 

```json
"url_pattern": "/__debug/test?userId=/{JWT.sub}"
```

```
[GIN] 2020/11/16 - 07:16:39 | 404 |         8.1µs |      172.17.0.1 | GET      "/__debug/test?userId=/5490"
[GIN] 2020/11/16 - 07:16:39 | 500 |    436.0351ms |      172.17.0.1 | GET      "/test"
```

**Comment by kpacha** [comment]
Nice catch! the patch should be released in v1.2.1

thanks for the heads up!

**Comment by MrSmoke** [comment]
legend :heart: thanks for the quick fix!
